pinot-tools/src/main/java/com/linkedin/pinot/tools/admin/command/AvroSchemaToPinotSchema.java

```diff
@@ -69,6 +69,11 @@
   public boolean execute() throws Exception {
     Schema schema = null;
 
+    if (_dimensions == null && _metrics == null) {
```

Comment:
The reason I didn't use the required attribute is in case someone wanted to create a table using only dimensions or only metrics